src/selectors/entities/common.js

```diff
@@ -13,7 +14,7 @@ export function getCurrentChannelId(state: GlobalState): string {
     return state.entities.channels.currentChannelId;
 }
 
-export function getMyChannelMemberships(state: GlobalState): string {
+export function getMyChannelMemberships(state: GlobalState): {[string]: ChannelMembership} {
```

Comment:
Not sure if I understand this correctly, but why not `{string: ChannelMembership}`?

Reply:
It saying that it is an object with string values as properties and channelMembership as values

Reply:
Thanks @enahum! Good to know, I initially thought that `[string]` is an "array of string values" but when I checked again array of strings is `Array<string>`.

Reply:
Yes, the problem is if you use {string: ChannelMembership} you are expressing that the objects have the key "string", and the value of type ChannelMemberships, in the other way you define "keys of type string" and values of type ChannelMemberships.